tinker/admin/redirects/models.py

```diff
@@ -4,19 +4,23 @@
 
 class BethelRedirect(db.Model):
     id = db.Column(db.Integer, primary_key=True)
-    from_path = db.Column(db.String(256), unique=True, default='no_path_provided')
+    from_path = db.Column(db.String(256), unique=True, default='no_path_provided',)
```

Comment:
is there a reason there is a comma at the end?

Reply:
Nope! Removed.